sentry/src/main/java/io/sentry/TracingOrigins.java

```diff
@@ -12,7 +12,7 @@ public final class TracingOrigins {
 
   public static boolean contain(final @NotNull List<String> origins, final @NotNull String url) {
     if (origins.isEmpty()) {
-      return true;
+      return false;
```

Comment:
Should we rename the class to `TracePropagationTargets`? I think it's only used by us, right?

Reply:
I think I'd vote for renaming it to be consistent, but up to you

Reply:
Renamed